.gitignore

```diff
@@ -10,3 +10,4 @@ Gemfile.lock
 .idea
 debug.log
 /test/db/*
+.env
```

Comment:
I know, I'm quite rigorous, but I think this isn't project related either!

There are three levels of ignoring in git:

1. `.gitignore`, corresponding to what the project want to ignore
2. `.git/info/exclude`, local to a projet and a specific person
3. `~/.gitignore` (configurable with `git config --global core.excludesFile path/to/file`) that is computer wide.

I think this is a good (3)!

See [git documentation](https://git-scm.com/docs/gitignore).

Reply:
Fixed